requirements.txt

```diff
@@ -1,4 +1,8 @@
--r docs/requirements.txt
+# Temporarily we had to remove support for the secure dependencies, because
+# readthedocs.org don't support pip8.
+# More info at https://github.com/rtfd/readthedocs.org/issues/1843
+
+-r docs/safe_requirements.txt
```

Comment:
I wanted to leave safe requirements for developers. I can remove safe_requirements file and just leave docs/requirements.txt if that's what you suggest.